xla/debug_options_flags.cc

```diff
@@ -443,6 +443,7 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
   opts.set_xla_gpu_dot_merger_threshold_mb(32);
   opts.set_xla_enable_fast_math(false);
   opts.set_xla_gpu_experimental_parallel_collective_overlap_limit(1);
+  opts.set_xla_gpu_experimental_parallel_async_compute_limit(2);
```

Comment:
i'd separate the compute limit change from the change to delay async start as they seems to be independent from each other.

Reply:
Done.